lsstswBuild.sh

```diff
@@ -63,7 +63,7 @@ print_error() {
 }
 
 fail() {
-    code=${2:1}
+    local code=${2:-1}
```

Comment:
Why does using local matter?  You're about to exit anyway, so if you clobber a global $code...so what?